pom.xml

```diff
@@ -0,0 +1,23 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
This is the maven project file. Read about its contents at [Maven Getting Started Guide](https://maven.apache.org/guides/getting-started/) (at least up to **How do I create a JAR and install it in my local repository?**)